ga/20.0.0.8/images.txt

```diff
@@ -0,0 +1,6 @@
+websphere-liberty:20.0.0.8-kernel              ../ga/20.0.0.8/kernel    ../ga/20.0.0.8/kernel/Dockerfile.ubuntu.ibmjava8
```

Comment:
Not sure if it's used for tag creation in Docker Hub or not, but these two tags should be `websphere-liberty:20.0.0.8-kernel-java8-ibmjava` and `websphere-liberty:20.0.0.8-full-java8-ibmjava` respectively